marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/system/createCustomRewriters.api

```diff
@@ -0,0 +1,4 @@
+{
+    "functionName": "createCustomRewriters",
+    "desc": "Creates custom rewriter modules for the staging and job app servers"
```

Comment:
I jumped the gun here a bit - 7334 is going to build on this and create the staging rewriter as well